Issue Template

**Description**
The current issue template for bug report has some information that is not required.

**Expected Behavior**
Remove "Meshery Version", "Kubernetes Version" and the "To Reproduce" section.
Also, fix the spelling of "Environment".

**Screenshots**
<img width="559" alt="Screenshot 2021-08-23 at 11 07 48 PM" src="https://user-images.githubusercontent.com/53532328/130492805-edbbbf7d-9739-4223-9aae-20c4e103ef12.png">


**Enviroment:**
- Host OS: 
- Meshery Version: 
- Kubernetes Version: 
- Browser: 

---

[Optional] **To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

[Optional] **Additional Context**
<!-- Add any other context about the problem here. -->

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

@sayantan1413 thanks for assigning me the issue! I am trying to clone the repository and trying to install the dependency: ```
```
F:\GnuWin32\bin\make.exe setup-libs
make: *** No rule to make target `setup-libs'.  Stop.
```
Can you please assist me on how I can fix it. Thanks in advance!

--- a/.github/ISSUE_TEMPLATE/bug_report.md
+++ b/.github/ISSUE_TEMPLATE/bug_report.md
@@ -14,20 +14,12 @@ assignees: ''
 **Screenshots**
 <!-- Add screenshots, if applicable, to help explain your problem. -->
 
-**Enviroment:**
+**Environment:**
 - Host OS: 
-- Meshery Version: 
-- Kubernetes Version: 
 - Browser: 
 
 ---
 
-[Optional] **To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
 
 [Optional] **Additional Context**
 <!-- Add any other context about the problem here. -->

